lib/OnboardingMessageProvider.jsm

```diff
@@ -51,11 +51,11 @@ function getPlaceholderTrailheadCard(id) {
     id: "ONBOARDING_2",
     template: "onboarding",
     content: {
-      title: `onboarding-card-${id}-title`,
-      text: `onboarding-card-${id}-text`,
+      title: `onboarding-${id}-title`,
```

Comment:
The existing onboarding.ftl file had "card" -title/text without a "-card-" so the m-c patch I have follows that pattern.